scripts/tf-validate.tfrc

```diff
@@ -0,0 +1,6 @@
+provider_installation { 
```

Comment:
prefer this approximation to override provider with code in master instead of doing multiple string replacements in the examples